js/src/views/Web/web.js

```diff
@@ -67,28 +64,44 @@ export default class Web extends Component {
       );
     }
 
+    const { dappsUrl } = this.context.api;
+    const { url } = this.state;
+    if (!url || !token) {
+      return null;
+    }
+
+    const parsed = parseUrl(url);
+    let host = parsed.host;
+    let path = parsed.path;
+    if (!host) {
+      host = parsed.path.split(pathSep).slice(0, 1);
+      path = parsed.path.split(pathSep).slice(1).join(pathSep);
```

Comment:
Isn't there missing the first `/` then ? Try `reddit.com/r/ethereum`

Also, what's the point of using `pathSep` ? Should always be `/` with URLs, no ?